### ISSUE TITLE
Add a secret type mirroring v128

### ISSUE THREAD

**Issue by CountBleck** [WebAssembly/constant-time#3]
A secret analog of v128 could perhaps be used for faster SIMD implementations of cryptographic primitives. Therefore, it might be a good idea to add one. Like the existing secret types, only a subset of the existing v128 instructions should be added for the secret variant.

**Comment by deian** [comment]
Yep! Will try to update the spec in the next couple of weeks and add this to it.

**Comment by tlively** [comment]
Are there constant time SIMD instructions in hardware? If not, constant time SIMD operations would have to be implemented in software and there would be no benefit to having a secret v128 type, IIUC.

**Comment by hovav** [comment]
It looks to me like Intel's [list of instructions with data-independent timing](https://www.intel.com/content/www/us/en/developer/articles/technical/software-security-guidance/resources/data-operand-independent-timing-instructions.html) includes some vector integer operations.

**Comment by garrettgu10** [comment]
[ARM's list](https://developer.arm.com/documentation/ddi0601/2020-12/AArch64-Registers/DIT--Data-Independent-Timing) contains SIMD instructions as well! :-)
